go/pkg/clickhouse/client.go

```diff
@@ -109,11 +131,15 @@ func New(config Config) (*clickhouse, error) {
 			Drop:          true,
 			BatchSize:     50_000,
 			BufferSize:    200_000,
-			FlushInterval: 60 * time.Second,
+			FlushInterval: 5 * time.Second,
 			Consumers:     2,
 			Flush: func(ctx context.Context, rows []schema.ApiRequest) {
 				table := "default.api_requests_raw_v2"
-				err := flush(ctx, conn, table, rows)
+				err := flush(ctx, conn, table, rows, flushCfg, WithInsertSettings(InsertSettings{
```

Comment:
this feels overly verbose, why don't we just apply the settings inside the flush method?
I don't think we'll ever want to use different settings per table

Reply:
same with the circuitbreaker and retry actually

we can simplify this a lot by just always using both retries and the circuitbreaker in our flush function